src/utils/api.js

```diff
@@ -6,18 +6,23 @@ import {
   deleteDoc,
 } from "firebase/firestore";
 import * as firebase from "firebase/app";
```

Comment:
imports everything from firebase? Now redundant as importing specific module needed in line 9. Individual modules should be imported as and when needed.